frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,27 @@
+import avatar from '~/assets/img/user-avatar.svg';
+import { Link } from '~/libs/components/link/link.jsx';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  user: unknown;
+};
+
+const Header: React.FC<Properties> = ({ user }) => (
+  <>
+    {user && (
+      <header className={styles.header}>
+        <Link to="/">
+          <span className={styles.logo}>WRITORIUM</span>
+        </Link>
+        <div className={styles['avatar-info']}>
+          <Link to="/profile">
+            <img src={avatar} alt="avatar" className={styles.avatar} />
```

Comment:
Let's create separate common component for avatar - as we may use it in different places throughout the app
it may take user name initials (first to letters of first/last name -- we will use it as placeholder when there is no avatar image applied) and `avatarUrl: string | null` 
In case if user has avatar uploaded - then we'd show it as image, in other case we'd show placeholder with initials
something like this
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/19575839/301d5d5f-e379-48fd-ba1a-6a845afb285a)

Reply:
Do the colors have to be the same for a non-existent avatar or can I use green for example?

Reply:
On daily you said that there's layout for non-existing avatar but there's no it yet, so can I leave for now something like this?
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/84803938/61f8b77f-43fb-4f4c-adb3-89c58abafa4a)

Reply:
So while the default user avatar layout isn't done, can I use something like this for now?

.default-avatar {
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 20px;
  font-weight: bold;
  background: lightblue;
  color: rgb(17, 159, 206);
}

Reply:
A little bit corrected:
```
/** @markup */
<div className={styles.defaultAvatar}>
  <span className={styles.initials}>{initials}</span>
</div>

/** @styling */
 /** colors and vars for them taken from Andrew Bobrovsky's message in slack */
.defaultAvatar {
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: #44996B; // --light-green
}
.initials {
  color: #2E453B; // --dark-green
  font-size: 20px;
  font-weight: 500;
  text-transform: uppercase;
}
```